.vscode/tasks.json

```diff
@@ -74,7 +74,10 @@
 				"$tsc",
 				"$eslint-stylish"
 			],
-			"group": "build",
+			"group": {
```

Comment:
What is this change?

Reply:
Ok these are just unnecessary . Committed them by mistake you can remove them

Reply:
Can you please do it